test/common/3scale_smtp.go

```diff
@@ -49,6 +49,10 @@ var (
 	originalPassword              = ""
 	originalPort                  = ""
 	originalUsername              = ""
+	original3ScalePassword        = ""
```

Comment:
hey, not sure if it's valid, but is there any chance these values will not be "" at the time when the test starts? and we are going to change them to "" possibly making this test destructive?

Reply:
We will always first path 3scale secret (this will populate these) and then reset it (using those to update the secret). To make it destructive we could just add an `if` statement switching off storing original details. I could hardcode them and skip the step of retrieving the original data, but if we will decide to change the default values in that secret - this test will modify the installation.

Reply:
Thanks Maks